### DISCUSSION TITLE
Why do the binaries have different checksums?

### DISCUSSION THREAD

**Discussion by eddmanx** [elishacloud/dxwrapper#315] [Q&A] [answered]
I thought the purpose of the dx7 and dx8 packages was to simply make it easier for the users, just copy over the files and play, but the dlls along with the ones from the regular package have different checksums.

Why is that?

**Comment by elishacloud** [comment, marked as answer]
I was using a different configuration for dx7 vs dx8 vs dxwrapper.  However, I updated the workflow to use the dxwrapper configuration to create the other artifacts,  So they should all have the same checksum in the latest build.
